python/ray/autoscaler/_private/vsphere/node_provider.py

```diff
@@ -336,76 +313,6 @@ def attach_tag(self, vm_id, resource_type, tag_id):
             logger.warning(f"Check that the tag is attachable to {resource_type}")
             raise e
 
-    def set_cloudinit_userdata(self, vm_id):
```

Comment:
We remove the functionalities of cloud init for now, because:
1. Our Packer project: https://github.com/vmware-ai-labs/vm-packer-for-ray, has already provided the functionality.
2. We have no doc to instruct the user about how to setup a user data for cloud init.